documentation-site/pages/blog/base-web-v11/index.mdx

```diff
@@ -0,0 +1,103 @@
+import Layout from '../../../components/layout';
+import { Meta } from '../../../components/blog';
+import Code from '../../../components/code';
+import metadata from './metadata.json';
+
+export default Layout;
+
+<Meta data={metadata} />
+
+# Changes in Base Web@v11
+
+The latest Base Web major release removes tech debt that has been deprecated for roughly two years.
+Many deprecated features have had warnings present at development-time and others will auto-fix
+warnings when you have our [eslint plugin](https://www.npmjs.com/package/eslint-plugin-baseui) installed.
+
+### `baseui/button`
+
+- The `minimal` option for the Button `kind` prop is deprecated in favor of `tertiary`.
+
+### `LocaleProvider`
+
+- The `timePickerAriaLabel` prop is removed in favor of `timePickerAriaLabel12Hour` or `timePickerAriaLabel24Hour`.
+
+### `baseui/modal`
+
+- The `Backdrop` override property is removed in favor of `DialogContainer`.
+- The `unstable_ModalBackdropScroll` prop is removed and become the default behavior.
+- The `autofocus` prop is removed in favor of `autoFocus`.
+
+### `baseui/phone-input`
+
+- The design has been updated to split the dial code selector into a separate control (`CountryPicker`). Until now, it was available as [`PhoneInputNext`](https://baseweb.design/components/phone-input/#updated-phone-input-design). This is now default and `PhoneInputNext` is removed.
+- The `CountrySelect` component is removed in favor of `CountryPicker`.
+
+### `baseui/spinner`
+
+- The design has been updated to use a fill without rounded corners. Until now, it was available now as [`StyledSpinnerNext`](https://baseweb.design/components/spinner/#spinner-next). This is now default and `StyledSpinnerNext` is removed.
+
+### `baseui/accordion`
+
+- The `renderPanelContent` prop is removed in favor of [`renderAll`](https://baseweb.design/components/accordion/#accordion-seo-friendly).
+
+### `baseui/popover`
```

Comment:
This didn't make it into the v11 release, and may never land. We can probably remove it from this post and also the roadmap until we decide to commit to changing it. There will be a lot of changes across several components.

Reply:
oh I meant to remove this from the blog post, left if on the roadmap page